docs/quick-start.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Quick Start
+description: Learn how to install Apify CLI, and how to create, run, and manage Actors through it.
+sidebar_label: Quick Start
+---
+
+Learn how to install Apify CLI, and how to create, run, and manage Actors through it.
+
+### Step 1: Installation
+
+You can install Apify CLI either using installation scripts:
+
+#### Preferred methods
+
+##### MacOS / Unix
+
+```bash showLineNumbers
+curl -fsSL https://apify.com/install-cli.sh | bash
+```
+
+##### Windows
+
+```bash showLineNumbers
+irm apify.com/install-cli.ps1 | iex
```

Comment:
shouldnt the URL be fully qualified?

Reply:
I just went off of bun's example, which doesn't have it (and it does work without it). That said, I'm fine with it being fully qualified too!

Reply:
Done